src/main/java/uk/gov/ons/census/action/builders/PrintFileDtoBuilder.java

```diff
@@ -17,7 +17,9 @@
           ActionType.P_RL_2RL1A,
           ActionType.P_RL_2RL2BA,
           ActionType.P_RL_1IRL1,
-          ActionType.P_RL_1IRL2B
+          ActionType.P_RL_1IRL2B,
+          ActionType.P_RL_1RL1B,
+          ActionType.P_RL_1RL2BB
```

Comment:
List should include P_RL_1RL4 - should be no UAC

Reply:
Nick yes good spot. I believe the code is P_RL_1RL4A (for some reason 4A is the northern ireland code) - have I got that right?

Reply:
yep, right 👍

Reply:
acually no, it's `P_RL_1RL4` like I said

Reply:
I've made the change so that the list now includes P_RL_1RL4.